ipyvolume/embed.py

```diff
@@ -140,63 +57,118 @@ def save_js(dirname, makedirs=True):
     src = os.path.join(dir_name_src, "index.js")
     shutil.copy(src, dst)
 
-def embed_html(filename, widgets, makedirs=True, copy_js=True,
-               drop_defaults=False, all=False, title="ipyvolume embed example", external_json=False,
-               indent=2,
-               template=template, template_options={"embed_url":ipywidgets.embed.DEFAULT_EMBED_SCRIPT_URL},
-               widget_view_template=widget_view_template, **kwargs):
-    """ output the the ipywidgets to a standalone html file
-    
-    :param filename: filepath to output to
-    :param widgets: list of ipywidget instances
-    :param makedirs: whether to make directories in the filename path, if they do not already exist
-    :param copy_js: whether to copy a local version of the ipyvolume javascript to the same folder
-    
+
+def save_requirejs(filepath='require.min.js', url="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js"):
+    """ download and save the require javascript to a local file
+
+    :type filepath: str
+    :type url: str
+    """
+    content = requests.get(url).content
+    with io.open(filepath, 'w', encoding='utf8') as f:
+        f.write(content.decode("utf8"))
+
+
+def save_font_awesome(dirpath='font-awesome', url="http://fontawesome.io/assets/font-awesome-4.7.0.zip"):
+    """ download and save the font-awesome package to a local folder
+
+    :type dirpath: str
+    :type url: str
+
     """
+    parentdirname = os.path.dirname(dirpath)
+    print("Downloading %s to %s" % (url, os.path.abspath(dirpath)))
     try:
-        widgets[0]
-    except (IndexError, TypeError):
-        widgets = [widgets]
-        
-    dir_name_dst = os.path.dirname(os.path.abspath(filename))
+        zip_folder = io.BytesIO(requests.get(url).content)
+        unzip = zipfile.ZipFile(zip_folder)
+        top_level_name = unzip.namelist()[0]
+        unzip.extractall(parentdirname)
+    except Exception as err:
+        raise IOError('Could not save: {0}\n{1}'.format(url, err))
+
+    if os.path.exists(dirpath):
+        shutil.rmtree(dirpath)
+    os.rename(os.path.join(parentdirname, top_level_name), dirpath)
```

Comment:
This should be included by webpack file (see https://github.com/maartenbreddels/ipyvolume/issues/41#issuecomment-322916169) , can you create an issue for this on ipywidgets? If fixed, we can delete this.